carve-bead.lic

```diff
@@ -0,0 +1,152 @@
+=begin
+  Carves a prayer bead from scratch.
```

Comment:
Ideally, a lot of this info would live in elanthipedia

Reply:
I can add a section for this script if we merge it